eservice/lib/libpdo_enclave/contract_worker.cpp

```diff
@@ -38,7 +38,9 @@ void ContractWorker::InitializeInterpreter(void)
     if (current_state_ == INTERPRETER_DONE)
     {
         if (interpreter_ == NULL) {
+            LOG_MEMORY;
             interpreter_ = pdo::contracts::CreateInterpreter();
```

Comment:
do we really need to include these? i can see that LOG_MEMORY would be useful, but it seems likely that we'll be using it for debugging purposes. i really don't want to see this on every invocation.

Reply:
I can remove it,
but the log is enabled only when `PDO_DEBUG_BUILD` is set.

Reply:
done

Reply:
precisely. and given that i always run with debug build... i really don't want to see these unless i'm debugging memory.

Reply:
I've removed it.